### PR TITLE
Preparations for version 1.6

### PR DESCRIPTION
**Pull request by cereda**
**TL;DR:** we ducks are not good at pull requests, branch management, rebases and other `git` operations. :duck: :grin: 

Let's prepare version 1.6 of our beloved `sillypage`! Things done so far:

- [x] Updated `sillywalk-map.pdf` per issue #10 (thanks Frank)
- [x] Updated `CHANGELOG` to include an entry for version 1.6
- [x] Updated `sillypage.dtx` (release version, date and copyright year)
- [x] Updated `sillypage.ins` (copyright year)

Things to do:
- [x] Update tests (@PhelypeOleinik could you take a look at it, please? :heart:)
- [x] Review this pull request (hint: let's ensure `l3build ctan` works :grin:)
- [x] Accept merge after this PR is reviewed
- [x] Remove the `sillymap-bw` branch
- [x] Create `v.16` tag and new release
- [x] Send update to CTAN

### CODE REVIEW

**Review by samcarter — APPROVED** [review]
Oh, a new version! Looks all good to me.  

@cereda Thanks a lot for putting it together!

**Review by PhelypeOleinik — APPROVED** [review]

**Comment by PhelypeOleinik** [comment]
It was a silly (pun very much intended :) error.  The PDF file changed and the value of the `id` shown by `graphics` (`\pdflastximage` in pdfTeX) changed.  Nothing to lose one's hair (or feathers) about :)

**Comment by cereda** [comment]
ooh :duck:
